packages/react-table/src/components/Table/utils/transformers.test.tsx

```diff
@@ -98,10 +38,6 @@ describe('Transformer functions', () => {
       };
       const returnedData = selectable('', { column, rowIndex: 0, rowData: { selected: true } } as IExtra);
       expect(returnedData).toMatchObject({ className: 'pf-c-table__check' });
```

Comment:
here you did delete the aspect of the test that verified that the selecting event handlers are getting triggered on click. Should we either make a new test to verify that?